dspace-server-webapp/src/main/resources/application.properties

```diff
@@ -130,3 +130,6 @@ spring.servlet.multipart.max-file-size = 512MB
 
 # Maximum size of a multipart request (i.e. max total size of all files in one request) (default = 10MB)
 spring.servlet.multipart.max-request-size = 512MB
+
+## Maximum page size per request (default = 1000)
+spring.data.rest.max-page-size=9999
```

Comment:
I agree. The main problem I have is that this number seems to be "default" for a lot of requests called from ds-angular (#8577 is one) so I use it. I think we really need a cleanup on the angular side since I guess they are trying to bypass the pagination entirely by setting a high ceiling.

Reply:
As @abollini notes above, this needs to be removed.  We cannot change this default value to be larger, as that could cause major performance issues for large sites.